lib/beetle/message.rb

```diff
@@ -218,6 +235,10 @@ def exceptions_limit_reached?
       @store.get(msg_id, :exceptions).to_i > exceptions_limit
     end
 
+    def exception_registered?
+      on_exceptions.nil? || @exception.class.in?(on_exceptions)
```

Comment:
Is this correct? Shouldn't it be false if `on_exceptions.nil?`?

Reply:
no actually not - when on_exceptions is not set it means all exception work.

the signature has obviously to change 😄 and the implementation as well (error subtypes mathcing and such)